### PR TITLE
add link to python api docs

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Add link to python api docs

image

## Testing notes:

1. run docs
2. click the link

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -112,6 +112,11 @@ const config = {
                         label: "REST API Docs",
                         position: "left",
                     },
+                    {
+                        href: "https://ynput.github.io/ayon-python-api/build/html/ayon_api.html",
```
Comment (robin-ynput) [inline website/docusaurus.config.js:116]: this is actually a temporary path, my work is not done yet.
To be confirmed but final location will actually be `https://ynput.github.io/ayon-python-api/`

**Review by robin-ynput — APPROVED** [review]
Thanks @MustafaJafar  ! I'll keep you posted as soon as it's live on my end

**Comment by MustafaJafar** [comment]
This PR is converted to draft till the python api doc link goes live.
Also, this PR might be closed in favor of https://github.com/ynput/ayon-documentation/pull/239. but, it's still up to the discussion.

**Comment by MustafaJafar** [comment]
Closing this PR in favor of the merged https://github.com/ynput/ayon-documentation/pull/239.

**Pull request closed without merging**